configure.py

```diff
@@ -171,16 +171,16 @@ def _gen_ios_cross_file(cfg):
         sha256="d7ed9c743c02469869f4bee7cb94641377b640676cb2a8a9a7955f2d07c4d8a6",
     ),
     ffmpeg=dict(
-        version="7.0",
+        version="7.1.1",
         url="https://ffmpeg.org/releases/ffmpeg-@VERSION@.tar.xz",
         dst_file="ffmpeg-@VERSION@.tar.xz",
-        sha256="4426a94dd2c814945456600c8adfc402bee65ec14a70e8c531ec9a2cd651da7b",
+        sha256="733984395e0dbbe5c046abda2dc49a5544e7e0e1e2366bba849222ae9e3a03b1",
     ),
     ffmpeg_Windows=dict(
-        version="7.0",
-        url="https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2024-04-30-12-51/ffmpeg-n@VERSION@-21-gfb8f0ea7b3-win64-gpl-shared-7.0.zip",
+        version="7.1.1",
+        url="https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2025-03-31-12-55/ffmpeg-n@VERSION@-5-g276bd388f3-win64-lgpl-shared-7.1.zip",
```

Comment:
I don't mind the switch from GPL to LGPL but it probably needs to be mentioned? (The consequence is a bunch of filters aren't built anymore, postproc goes away, and I think an optim in the flac encoder

Reply:
Mentionned in the commit message.